examples/auth/index.js

```diff
@@ -81,53 +73,48 @@ function restrict(req, res, next) {
   }
 }
 
-app.get('/', function(req, res){
+// Routes
+app.get('/', (req, res) => {
   res.redirect('/login');
 });
 
-app.get('/restricted', restrict, function(req, res){
-  res.send('Wahoo! restricted area, click to <a href="/logout">logout</a>');
+app.get('/restricted', restrict, (req, res) => {
+  res.send(`Wahoo! restricted area, click to <a href="/logout">logout</a>`);
 });
 
-app.get('/logout', function(req, res){
-  // destroy the user's session to log them out
-  // will be re-created next request
-  req.session.destroy(function(){
+app.get('/logout', (req, res) => {
+  req.session.destroy(() => {
     res.redirect('/');
   });
 });
 
-app.get('/login', function(req, res){
+app.get('/login', (req, res) => {
   res.render('login');
 });
 
-app.post('/login', function (req, res, next) {
-  authenticate(req.body.username, req.body.password, function(err, user){
-    if (err) return next(err)
+app.post('/login', (req, res) => {
+  authenticate(req.body.username, req.body.password, (err, user) => {
+    if (err) {
+      console.error('Authentication error:', err);
+      req.session.error = 'Internal server error during authentication.';
+      return res.redirect('/login');
+    }
+
     if (user) {
-      // Regenerate session when signing in
-      // to prevent fixation
-      req.session.regenerate(function(){
-        // Store the user's primary key
-        // in the session store to be retrieved,
-        // or in this case the entire user object
+      req.session.regenerate(() => {
         req.session.user = user;
-        req.session.success = 'Authenticated as ' + user.name
-          + ' click to <a href="/logout">logout</a>. '
-          + ' You may now access <a href="/restricted">/restricted</a>.';
+        req.session.success = `Authenticated as ${user.name}. Click to <a href="/logout">logout</a>. You may now access <a href="/restricted">/restricted</a>.`;
         res.redirect('back');
       });
     } else {
-      req.session.error = 'Authentication failed, please check your '
-        + ' username and password.'
-        + ' (use "tj" and "foobar")';
+      req.session.error = 'Authentication failed. Please check your username and password.';
       res.redirect('/login');
     }
   });
 });
 
-/* istanbul ignore next */
```

Comment:
Please don't remove this comment as it will have a negative impact in our tests coverage reports